cmake/modules/AddSwift.cmake

```diff
@@ -1826,7 +1837,10 @@ function(add_swift_target_library name)
     endif()
 
     # For each architecture supported by this SDK
-    foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
+    set(sdk_supported_archs
+      ${SWIFT_SDK_${sdk}_ARCHITECTURES}
+      ${SWIFT_SDK_${sdk}_MODULE_ARCHITECTURES})
```

Comment:
Should this be uniqued?

Reply:
I've gone ahead an added a `list(REMOVE_DUPLICATES ...)` call just in case.